charts/jaeger/README.md

```diff
@@ -170,24 +170,26 @@ collector:
       subPath: ""
       configMap: jaeger-tls
       readOnly: true
-```
+spark:
+  enabled: true
+  cmdlineParams:
+    java.opts: "-Djavax.net.ssl.trustStore=/tls/trust.store -Djavax.net.ssl.trustStorePassword=changeit"
+  extraConfigmapMounts:
+    - name: jaeger-tls
+      mountPath: /tls
+      subPath: ""
+      configMap: jaeger-tls
+      readOnly: true
 
-Content of the `jaeger-tls-cfgmap.yaml` file:
+```
 
-```YAML
-apiVersion: v1
-kind: ConfigMap
-metadata:
-  name: jaeger-tls
-data:
-  es.pem: |
-    -----BEGIN CERTIFICATE-----
-    <CERT>
-    -----END CERTIFICATE-----
+Generate configmap jaeger-tls:
```

Comment:
Is it not better to put this data in a secret rather than a cm?

Reply:
For my use case the configmap is ok, because it is only the certificate, not the private key.

Reply:
But with extraSecretMounts it is also possible to use a secret.